deployment/docker/Makefile

```diff
@@ -1,33 +1,38 @@
 # All these commands must run from repository root.
 
+DOCKER_REGISTRY ?= docker.io
 DOCKER_NAMESPACE ?= victoriametrics
 
-ROOT_IMAGE ?= alpine:3.20.0
-ROOT_IMAGE_SCRATCH ?= scratch
+ROOT_IMAGES ?= alpine:3.20.0 -scratch
 CERTS_IMAGE := alpine:3.20.0
 
 GO_BUILDER_IMAGE := golang:1.22.4-alpine
 BUILDER_IMAGE := local/builder:2.0.0-$(shell echo $(GO_BUILDER_IMAGE) | tr :/ __)-1
-BASE_IMAGE := local/base:1.1.4-$(shell echo $(ROOT_IMAGE) | tr :/ __)-$(shell echo $(CERTS_IMAGE) | tr :/ __)
+BASE_IMAGE := local/base:1.1.4-$(shell echo $(CERTS_IMAGE) | tr :/ __)-$(shell echo $(CERTS_IMAGE) | tr :/ __)
```

Comment:
This image tag is supposed to be persistent in case the root image for the build is not changed, so it seems like it should use `ROOT_IMAGES`.
If the value does not match the tag naming requirements it can use sha of an image name.